### PR TITLE
Namedtuple return for solve, slogdet, sort, topk

### PR DESCRIPTION
**Pull request by zasdfgbnm**
More ops for https://github.com/pytorch/pytorch/issues/394. ~~Also need to rebase after landing #16186, because we need to update the whitelist of the new unit test added in #16186.~~

cc: @ezyang

### CODE REVIEW

File: aten/src/ATen/native/native_functions.yaml [hunk 1]
```diff
@@ -1890,7 +1890,7 @@
   variants: function, method
   device_guard: False
 
-- func: slogdet(Tensor self) -> (Tensor, Tensor)
+- func: slogdet(Tensor self) -> (Tensor sign, Tensor log)
```
Comment (vishwakftw) [inline aten/src/ATen/native/native_functions.yaml]: Maybe `logabsdet` is a better alternative?

Comment (zasdfgbnm) [reply aten/src/ATen/native/native_functions.yaml]: @vishwakftw I agree, just renamed.

File: aten/src/ATen/native/native_functions.yaml [hunk 2]
```diff
@@ -4004,21 +4004,21 @@
     CPU: median_cpu
     CUDA: median_cuda
 
-- func: sort(Tensor self, int dim=-1, bool descending=False, *, Tensor(a!) values, Tensor(b!) indices) -> (Tensor(a!), Tensor(b!))
+- func: sort(Tensor self, int dim=-1, bool descending=False, *, Tensor(a!) values, Tensor(b!) indices) ->(Tensor(a!) values, Tensor(b!) indices)
```
Comment (ezyang) [inline aten/src/ATen/native/native_functions.yaml]: Probably didn't want to drop the space here?

**Review by ezyang — APPROVED** [review]

**Pull request closed without merging**